options/flags.go

```diff
@@ -78,6 +80,8 @@ const (
 	FlagCommitDepth = "commit-depth"
 
 	FlagProbes = "probes"
+
+	FlagCombinedOutput = "combined"
```

Comment:
is this intended to be only change the behavior if `--format=default`? Currently when I run something like this, I get the "combined" output instead of "json"
```
go run main.go --org=ossf-tests --format json --combined
```

Or should this be it's own format value? e.g. 
```
go run main.go --org=ossf-tests --format combined
```